### PR TITLE
chore(github): add contributing guidelines

### PR DESCRIPTION
**Pull request by vitorfec**
**What type of PR is this?**
Chore

**What this PR does / why we need it:**
To allow contributions and keep standards around the charts

**Which issue(s) this PR fixes:**
Closes [CORE-1024](https://cloudmobility.atlassian.net/browse/CORE-1024)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+### Documentation Requirements
+
+- A chart's `README.md` must include configuration options.
+- A chart's `NOTES.txt` must include relevant post-installation information.
+- The title of the PR starts with chart name (e.g. `[cloudmobility/chart]`)
+
+### PR Approval and Release Process
+
+1. Changes are automatically linted and tested using the [`ct` tool](https://github.com/helm/chart-testing) as a [GitHub action](https://github.com/helm/chart-testing-action). Those tests are based on `helm install`, `helm lint` and `helm test` commands and provide quick feedback about the changes in the PR. For those tests, the chart is installed on top of [kind](https://github.com/kubernetes-sigs/kind) and this step is not blocking (as opposed to 3rd step).
+2. Changes are manually reviewed by Bitnami team members.
```
Comment (NMFR) [inline CONTRIBUTING.md:29]: "Bitnami"?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+### Documentation Requirements
+
+- A chart's `README.md` must include configuration options.
+- A chart's `NOTES.txt` must include relevant post-installation information.
+- The title of the PR starts with chart name (e.g. `[cloudmobility/chart]`)
+
+### PR Approval and Release Process
+
+1. Changes are automatically linted and tested using the [`ct` tool](https://github.com/helm/chart-testing) as a [GitHub action](https://github.com/helm/chart-testing-action). Those tests are based on `helm install`, `helm lint` and `helm test` commands and provide quick feedback about the changes in the PR. For those tests, the chart is installed on top of [kind](https://github.com/kubernetes-sigs/kind) and this step is not blocking (as opposed to 3rd step).
```
Comment (NMFR) [inline CONTRIBUTING.md:28]: Don't think we have linting currently. But i guess we will add it soon.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+2. Changes are manually reviewed by Bitnami team members.
+3. Once the changes are accepted, the PR is tested (if needed) into the Bitnami CI pipeline, the chart is installed and tested (verification and functional tests) on top of different k8s platforms.
```
Comment (NMFR) [inline CONTRIBUTING.md:30]: "Bitnami"?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+4. When the PR passes all tests, the PR is merged by the reviewer(s) in the GitHub `master` branch.
+5. Then our CI/CD system is going to push the chart to the Helm registry including the recently merged changes and also the latest images and dependencies used by the chart. The changes in the images will be also committed by the CI/CD to the GitHub repository, bumping the chart version again.
+
+***NOTE***: Please note that, in terms of time, may be a slight difference between the appearance of the code in GitHub and the chart in the registry.
+
+[Here](https://docs.bitnami.com/kubernetes/faq/get-started/understand-charts-release-process/) you can find more information about this process.
```
Comment (NMFR) [inline CONTRIBUTING.md:36]: linking to bitnami docs?

**Comment by NMFR** [comment]
Ahh, sorry only noticed it was draft after reviewing. I received the email about the PR so i just started reviewing.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+1. Fork this repository, develop, and test your changes.
+2. Submit a pull request.
+
+***NOTE***: To make the Pull Requests' (PRs) testing and merging process easier, please submit changes to multiple charts in separate PRs.
+
+### Technical Requirements
+
+When submitting a PR make sure that it:
+- Must follow [Helm best practices](https://helm.sh/docs/chart_best_practices/).
+- Any change to a chart requires a version bump following [semver](https://semver.org/) principles.
```
Comment (tiagomichaelsousa) [inline CONTRIBUTING.md:18]: can we add [conventional commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/) to the list ? if we use conventional commits maybe we can add automatics changelogs for each chart in the future

**Review by tiagomichaelsousa — APPROVED** [review]

**Review by NMFR — APPROVED** [review]

**Pull request merged**